### PR TITLE
update og tag snapshotting for react rendering

### PR DESCRIPTION
**Pull request by stanleyzhengnyc**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+
+// render(
```
Comment (jtmckibb) [inline src/index.tsx:9]: ?

File: src/index.tsx [hunk 2]
```diff
@@ -1,16 +1,31 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { hydrate, render } from "react-dom";
```
Comment (jtmckibb) [inline src/index.tsx:2]: these should be single quotes

File: src/index.tsx [hunk 3]
```diff
@@ -1,16 +1,31 @@
...
-);
+
+// render(
+//   <React.StrictMode>
+//     <App />
+//   </React.StrictMode>,
+//   document.getElementById('root')
+// )
+
+const rootElement = document.getElementById("root");
```
Comment (jtmckibb) [inline src/index.tsx:16]: single quotes

**Comment by stanleyzhengnyc** [comment]
@jtmckibb addressed quotes and commented out portion

**Pull request merged**